core/create_readme.js

```diff
@@ -78,25 +78,27 @@ ${markdown}
 		const submissions = readme_data.submissions;
 
 		let output = `# theClubhou.se Project Euler Collaboration Repository
-## Submissions Leaderboard
+\n## Submissions Leaderboard
 `;
 		const clean = process_submissions(submissions);
 
+		let cnt = 1
 		for ( const user in clean.leaderboard ) {
-			output += `1. @${user} **${clean.leaderboard[user]}** entries\n`;
+			output += `${cnt}. @${user} **${clean.leaderboard[user]}** entries\n`;
+			cnt++;
 		}
 
-		output += '\n# Popular Languages\n';
+		output += '\n## Popular Languages\n';
 
 		const languages = Object.keys(clean.languages)
 			.sort((a,b) => clean.languages[b].count - clean.languages[a].count);
 		for ( let i = 0; i < languages.length; ++i ) {
-			output += `1. **${languages[i]}** with **${clean.languages[languages[i]].count}** entries [leader: @${clean.languages[languages[i]].max_user} - ${clean.languages[languages[i]].max}]\n`;
+			output += `${i+1}. **${languages[i]}** with **${clean.languages[languages[i]].count}** entries [leader: @${clean.languages[languages[i]].max_user} - ${clean.languages[languages[i]].max}]\n`;
```

Comment:
Boo... :D We are software engineers and start counting from "0" ;)

Reply:
You had "1" for every line!